conf.py

```diff
@@ -170,3 +191,9 @@
 tags_badge_colors = {
     "*": "primary",
 }
+
+# Sitemap settings
+
+sitemap_url_scheme = "{link}"
+sitemap_locales = [None]
+sitemap_filename = "sitemap.xml"
```

Comment:
I think you can remove the settings which are set to their defaults:

```suggestion
sitemap_locales = [None]
```